FWCore/Framework/test/stubs/TestOutputModule.cc

```diff
@@ -20,6 +20,8 @@
 
 using namespace edm;
 
+typedef detail::TriggerResultsBasedEventSelector::handle_t Trig;
```

Comment:
How about `using` instead?

Reply:
Done. Thanks, that is better. I had just copied that line and a little function up from the legacy base class without modifying them or really thinking  about it. It was not in the 'one' base class.